From: Michael Froman <mfroman@mozilla.com>
Date: Tue, 1 Jul 2025 18:38:00 -0500
Subject: Bug 1974813 - abseil-cpp - remove prevent_unsafe_narrowing build
 config since chromium/build now automatically includes it

Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/af60dbfe1bbea15ac5102a8ca90b9f6624ee151c
---
 abseil-cpp/absl.gni | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/abseil-cpp/absl.gni b/abseil-cpp/absl.gni
index f13ae7919ca..af7f68f9c85 100644
--- a/abseil-cpp/absl.gni
+++ b/abseil-cpp/absl.gni
@@ -38,6 +38,10 @@ template("absl_source_set") {
         "//abseil-cpp:absl_define_config",
       ]
 
+      if (moz_webrtc_build) {
+        configs -= [ "//chromium/build/config/compiler:prevent_unsafe_narrowing" ]
+      }
+
       if (!defined(defines)) {
         defines = []
       }
